### ISSUE TITLE
EOFError: read end of file

### ISSUE THREAD

**Issue by davidanthoff** [JuliaEditorSupport/julia-vscode#633]
On 0.11.0-beta.2

````
EOFError:
   at read (iobuffer.jl207)
   at read(::Base.PipeEndpoint, ::Type{UInt8}) (stream.jl825)
   at read(::Base.Process, ::Type{UInt8}) (io.jl229)
   at deserialize (Serialization.jl731)
   at deserialize(::Base.Process) (Serialization.jl718)
   at request(::SymbolServerProcess, ::Symbol, ::Nothing) (./scripts/languageserver/packages/SymbolServer/src/SymbolServer.jl39)
   at get_installed_packages_in_env(::SymbolServerProcess) (./scripts/languageserver/packages/SymbolServer/src/SymbolServer.jl107)
   at getstore(::SymbolServerProcess) (./scripts/languageserver/packages/SymbolServer/src/SymbolServer.jl66)
   at run(::LanguageServerInstance) (./scripts/languageserver/packages/LanguageServer/src/languageserverinstance.jl34)
   at top-level scope (./scripts/languageserver/main.jl26)
   at include (boot.jl326)
   at include_relative(::Module, ::String) (loading.jl1038)
   at include(::Module, ::String) (sysimg.jl29)
   at exec_options(::Base.JLOptions) (client.jl267)
   at _start() (client.jl436)
````

**Comment by davidanthoff** [comment]
I think what we need to do here is modify our telemetry story a bit: I assume the underlying symbol server process is crashing. So what we should do is to a) catch that and then b) send the content of the error stream of the symbol server process with our telemetry data.

I'm moving this to v0.11.1 because this is too complicated to fix for v0.11.0.

**Comment by davidanthoff** [comment]
Closing for now because we need more info, plus we should get a more recent telemetry crash report if this is still an issue.
